discrete_optimization/coloring/solvers/greedy_coloring.py

```diff
@@ -72,7 +72,7 @@ def __init__(
             params_objective_function=params_objective_function,
         )
 
-    def solve(self, **kwargs):
+    def solve(self, **kwargs) -> ResultStorage:
```

Comment:
Should have been done in another PR ;p  (and for some reason, this broke mypy ?!)